aurora-mysql-plugin/src/test/java/io/cdap/plugin/aurora/mysql/AuroraMysqlPluginTestBase.java

```diff
@@ -56,6 +58,7 @@
 import javax.sql.rowset.serial.SerialBlob;
 
 public abstract class AuroraMysqlPluginTestBase extends DatabasePluginTestBase {
+  private static final Logger logger = LoggerFactory.getLogger(AuroraMysqlPluginTestBase.class);
```

Comment:
style: private static final variable names should be all caps LOGGER

Reply:
done:)